lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -739,32 +739,51 @@ class ExprEmitter : public ExprVisitor<ExprEmitter, SubExprInfo>,
   SubExprInfo emitBitSelect(Value operand, unsigned hiBit, unsigned loBit);
 
   SubExprInfo emitBinary(Operation *op, VerilogPrecedence prec,
```

Comment:
We'd like to eventually remove the FIRRTL stuff from this file, and the RTL and FIRRTL ops are going to continue diverging.  Would it make sense to proactively fork things like `emitBinary` into an `emitBinary` (for RTL) and `emitFIRRTLBinary` function?  The later one can eventually be sloughed off.

Reply:
The reason I ask is that the "strict sign" idea makes sense for FIRRTL but not RTL, and the opForceSign bit (which would be better named "forceSigned") makes sense for RTL.

Reply:
It's pretty close to that now.  RTL sets the forced bit, firrtl never does.  emit essentially checks if it is emitting firrtl and does something extra in that case which would go away when firrtl does.

Reply:
Ok, I'd still prefer the split because it makes it easier to reason about what the end state (for RTL) will look like, without the complexity of the FIRRTL path mixed in.  If you strongly prefer this approach then that is ok with me, we're just talking about an intermediate period either way.